src/autocomplete/StreamAutocomplete.js

```diff
@@ -34,6 +34,7 @@ class StreamAutocomplete extends PureComponent<Props> {
     return (
       <Popup>
         <FlatList
+          nestedScrollEnabled
```

Comment:
Huh interesting. Are you finding that this makes a difference in the behavior you see?

The [docs say](https://reactnative.dev/docs/scrollview#nestedscrollenabled-android) this defaults to true, so passing it shouldn't be needed.

They also a bit concerningly say:
> Enables nested scrolling for Android API level 21+.

which makes me worry about whether the issue you were seeing will be there on iOS, when we add sharing support there (which I'd expect to use just about exactly the same JS code.) OTOH reportedly it works on iOS anyway:
https://stackoverflow.com/questions/59828595/react-native-what-does-nestedscrollenabled-do
On the other other hand, the people on that thread found that it worked on Android already without the flag, so that's different from what it sounds like you were seeing.

Reply:
Strangely yes, the change in this commit does make a difference. I have added it in 3 places and all of them are relevant for scrolling to work properly (scrolling of stream selection popup and topic selection popup) inside the sharing ui.

Reply:
So be it, then. Thanks!

I added a few words about this in this commit message, including a link to this thread.